src/native/eventpipe/ep-buffer-manager.c

```diff
@@ -1145,8 +1145,12 @@ ep_buffer_manager_write_all_buffers_to_file_v4 (
 
 	*events_written = false;
 
+	ep_rt_thread_session_state_list_iterator_t thread_session_state_list_iterator = ep_rt_thread_session_state_list_iterator_begin (&buffer_manager->thread_session_state_list);
```

Comment:
`thread_session_state_list_iterator` is declared here but not used before it's assigned again at line 1205